### PR TITLE
Added 'dcos task ls'.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

File: pkg/cmd/task/task_ls.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+			}
+
+			allTasks, err := mesosClient.Tasks()
+			if err != nil {
+				return err
+			}
+
+			var tasks []lib.Task
+			for _, t := range allTasks {
+				if strings.Contains(t.TaskID.Value, args[0]) {
```
Comment (bamarni) [inline pkg/cmd/task/task_ls.go]: Can we have the same task ID matching logic than the current CLI?

> A full task ID, a partial task ID, or a Unix shell wildcard pattern (eg. 'my-task*').

File: pkg/cmd/task/task_ls.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+				}
+			}
+
+			for i, t := range tasks {
+				if len(tasks) > 1 {
+					fmt.Fprintln(ctx.Out(), "===> "+t.TaskID.Value+" <===")
+				}
+				taskPath := "/frameworks/" + t.FrameworkID.Value + "/executors/" + executorIDs[i] + "/runs/" + containerParentIDs[i]
+				for key, value := range agentsPaths[t.AgentID.Value] {
+					if strings.Contains(key, taskPath) {
```
Comment (bamarni) [inline pkg/cmd/task/task_ls.go]: Should this be `HasPrefix`?

File: pkg/cmd/task/task_ls.go [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+				}
+			}
+
+			for i, t := range tasks {
+				if len(tasks) > 1 {
+					fmt.Fprintln(ctx.Out(), "===> "+t.TaskID.Value+" <===")
+				}
+				taskPath := "/frameworks/" + t.FrameworkID.Value + "/executors/" + executorIDs[i] + "/runs/" + containerParentIDs[i]
+				for key, value := range agentsPaths[t.AgentID.Value] {
+					if strings.HasSuffix(key, taskPath) {
```
Comment (bamarni) [inline pkg/cmd/task/task_ls.go:93]: Should it really be suffix and not prefix?

Comment (armandgrillet) [reply pkg/cmd/task/task_ls.go:93]: Yes, I've manually tested and the function to use in order to get the same behavior as in the Python CLI is to use `HasSuffix`.

**Review by bamarni — APPROVED** [review]

**Pull request merged**